packages/account-v2/src/modules/webpack.config.ts

```diff
@@ -0,0 +1,159 @@
+import path from 'path';
```

Comment:
just a suggestion: we may split config, just let it looking more concise :)

Reply:
Having the config in one place let's you work easier with it, I will look into it once we have this finalized.